### PR TITLE
Create ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre

Ticket: [MTM-58509](https://cumulocity.atlassian.net/browse/MTM-58509)
Original PR: [5975](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5975)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Create a type dashboard list view (#5975)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add dashboard list view
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58509
+version: 1020.0.0
+---
+Create a type dashboard list view (#5975)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:17]: The Claude AI suggested to use the following body:
```suggestion
To provide users with an overview of their dashboards, a new list view has been added. The dashboard list view displays all dashboards in a tabular format, showing relevant information such as the dashboard name, description, owner, and last modification date. This change enhances the user experience by making it easier to browse, search and manage multiple dashboards in one central place.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md (hunk 1 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:3]:
```suggestion
title: Add Dashboard manager feature with list of type dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:17]: Add Dashboard manager view. For now it contains only list of device type dashboards. It aggregates all dashboards that has device type assigned, it allows to remove these dashboards and it is also possible to view all the devices that has access to particular dashboard from list. User can edit dashboard by navigating to device view.

Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:17]:
```suggestion
Add Dashboard manager view. For now it contains only list of device type dashboards. It aggregates all dashboards that has device type assigned, it allows to remove these dashboards and it is also possible to view all the devices that has access to particular dashboard from list. User can edit dashboard by navigating to device view.
```

**Review by jdreSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58509
+version: 1020.0.0
+---
+Add Dashboard manager view. For now it contains only list of device type dashboards. It aggregates all dashboards that has device type assigned, it allows to remove these dashboards and it is also possible to view all the devices that has access to particular dashboard from list. User can edit dashboard by navigating to device view.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:17]:
```suggestion
A dashboard manager view has been added. This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. Dashboards can be removed via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Add Dashboard manager feature with list of type dashboards
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:3]:
```suggestion
title: New dashboard manager with list of device type dashboards
```

**Comment by BeateRixen** [comment]
@rahultsag Please review and approve

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: New dashboard manager with list of device type dashboards
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:3]: @janhommes  I assume that documentation will also be added shortly?

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58509
+version: 1020.0.0
+---
+A dashboard manager view has been added. This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. Dashboards can be removed via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view.
```
Comment (rahultsag) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:17]: Proposing this description : A dashboard manager view has been added to streamline the management of device type dashboards. This view aggregates and displays a list of all dashboards assigned to specific device types. From this list, dashboard templates linked to device types can be viewed ,edited by navigating to a particular instance, and templates can also be deleted if needed.

**Review by rahultsag — CHANGES_REQUESTED** [review]
Proposing this description : A dashboard manager view has been added to streamline the management of device type dashboards. This view aggregates and displays a list of all dashboards assigned to specific device types. From this list, dashboard templates linked to device types can be viewed ,edited by navigating to a particular instance, and templates can also be deleted if needed.

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md (hunk 7 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md:17]:
```suggestion
A dashboard manager view has been added to streamline the management of device type dashboards. This view aggregates and displays a list of all dashboards assigned to specific device types. From this list, dashboard templates linked to device types can be viewed, edited by navigating to a particular instance and templates can also be deleted if needed.
```

Comment (jdreSAG) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-create-a-type-dashboard-list-view-5975.md]: @rahultsag  your description is commited now

**Comment by BeateRixen** [comment]
@rahultsag Please approve.

**Review by rahultsag — APPROVED** [review]

**Pull request merged**